### ISSUE TITLE
Firefox 44 for Android not installing

### ISSUE THREAD

**Issue by jefft0** [lidel/ipfs-firefox-addon#92]
I have Firefox 44 on Android 4.4.2.(KitKat). When I try to install from addons.mozilla.org, it says "Downloading add-on" but it doesn't install. (Is the add-on supported on Android?)

**Comment by lidel** [comment]
Good question. Addon was not tested on Android and probably should be disabled for that platform. 

I tried to search for `ipfs`  at `addons.mozilla.org` from my mobile (Android 6) but got no results (expected). 

Then I opened [addon page]https://addons.mozilla.org/en-US/firefox/addon/ipfs-gateway-redirect/)  directly and I see green install button which when clicked displays "Downloading add-on" but does nothing.

**tl;dr** It seems to be a bug at AMO site or my mistake during last submission -- there should be no install button for Android. I'll double check it during next release. Closing for now.

If you would like to see Android support at some point in future, please open a new Issue for that.
